### PR TITLE
Fix dependency_management.rst typo

### PR DESCRIPTION
**Pull request by cnpryer**

### CODE REVIEW

File: docs/userguide/dependency_management.rst [hunk 1]
```diff
@@ -173,7 +173,7 @@ The environmental markers that may be used for testing platform types are
 detailed in :pep:`508`.
 
 .. seealso::
-   If  environment markers are not enough an specific use case,
+   If  environment markers are not enough of a specific use case,
```
Comment (cnpryer) [inline docs/userguide/dependency_management.rst:176]: Maybe something like this was intended?
```suggestion
   If  environment markers aren't sufficient,
```

Comment (abravalheri) [reply docs/userguide/dependency_management.rst:176]: Thank you very much Chris.
I think the original phrase was missing a `for`: `... for a specific use case`.

I understand that the second suggestion is more concise and clean.

However, I do like the emphasis that the wrapper solution should only be used for a small amount of specific use cases... What do you think?

Comment (cnpryer) [reply docs/userguide/dependency_management.rst:176]: Got it. What about "Alternatively, [the backend wrapper] can be used for specific use cases where markers aren't sufficient"

Comment (cnpryer) [reply docs/userguide/dependency_management.rst]: Done! 66d7d85cf3cbe58aa2c2d56bce07585cf93def37

**Comment by abravalheri** [comment]
Thank you very much!

**Pull request merged**